examples/s/d/nn/xcase/realization-0/iter-0/fmuconfig/output/global_variables.yml

```diff
@@ -17,7 +17,7 @@ masterdata:
       uuid: ad214d85-dac7-19da-e053-c918a4889309
     stratigraphic_column:
       identifier: DROGON_2020
-      uuid: some-unique-id-to-be-provided-by-smda
```

Comment:
Unsure how this passed before 🤔

Reply:
These are the examples, they aren't validated I believe.